aztec/src/main/kotlin/org/wordpress/aztec/util/SpanWrapper.kt

```diff
@@ -17,7 +23,18 @@ class SpanWrapper<T>(var spannable: Spannable, var span: T) {
 
     var flags: Int
         get() { return spannable.getSpanFlags(span) }
-        set(flags) { spannable.setSpan(span, start, end, flags) }
+        set(flags) {
+            // Do not set the span if it's a PARAGRAPH that doesn't start at paragraph boundary
+            // Copied from SpannableStringBuilder that throws an exception in this case.
+            val flagsStart = flags and START_MASK shr START_SHIFT
+            if (!isInvalidParagraph(start, flagsStart)) {
```

Comment:
Since we're checking for the correct start bound of a paragraph, should we also check for the correct end (the last char is a newline or end == text.length)?

Reply:
Right, i've omitted that check on purpose, since we're not seeing crashes for it, but now that I think more on it I agree with you, better to have that check in the code.